mikeio/dfs0.py

```diff
@@ -279,6 +287,7 @@ def create(
 
         delete_value = dfs.FileInfo.DeleteValueFloat
 
+        data = data.copy()
         for i in range(n_items):
             d = data[i].copy()
             d[np.isnan(d)] = delete_value
```

Comment:
Alternative used previously
```suggestion
        nan_indices = np.isclose(data, self._dfs.FileInfo.DeleteValueFloat, atol=1e-36)
        data[nan_indices] = np.nan
```